flocker/node/agents/test/test_blockdevice_manager.py

```diff
@@ -157,3 +167,102 @@ def test_formatted_bad_type(self):
         blockdevice = self._get_free_blockdevice()
         with self.assertRaisesRegexp(MakeFilesystemError, blockdevice.path):
             self.manager_under_test.make_filesystem(blockdevice, 'myfakeyfs')
+
+    def test_bind_mount(self):
+        """
+        Files created in a bind mount are visible in the original folder and
+        vice versa.
+        """
+        src_directory = self._get_directory_for_mount()
+        target_directory = self._get_directory_for_mount()
+        self.manager_under_test.bind_mount(src_directory, target_directory)
+        for create, view in [(target_directory, src_directory),
+                             (src_directory, target_directory)]:
+            filename = str(uuid4())
+            new_file = create.child(filename)
+            new_file.touch()
+            self.expectThat(view.child(filename).exists(), Equals(True),
+                            'Created file not visible through bind mount.')
```

Comment:
I thought about that, couldn't decide if ".exists(), Equals(True)" or ".path, FileExists()" was better.

I'll switch to your way though because twisted's ".exists()" is subtly broken, and I can remove a massive comment if I don't use it :)